CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ All notable changes to this project will be documented in this file.
 
 ### Changed
 
+-   #1484 : Improving handling of `DottedName` in `_assign_lhs_variable`.
```

Comment:
I think you can add
```
-    #1484 : Using scope for classes to avoid name clashes 
```

This PR fixes your test file

Reply:
Beware when you merge. The version was just updated to 1.9.1 so you will need to move your comments to a newer version